### PR TITLE
Prevent upgrades on win32

### PR DESCRIPTION
**Pull request by tejohnso**
@ahmedalsudani @fjvallarino please review

### CODE REVIEW

File: versions-config/electron-remote-components-win-32.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "ForceStable": false,
-  "LatestRolloutPercent": 10,
+  "LatestRolloutPercent": 0,
   "InstallerElectronVersion": "2016.3.15.15.34",
```
Comment (ahmedalsudani) [inline versions-config/electron-remote-components-win-32.json]: @tejohnso we're not reverting the version to the last one?

Comment (tejohnso) [inline versions-config/electron-remote-components-win-32.json]: @ahmedalsudani If the rollout percentage isn't met the version won't matter.

Comment (ahmedalsudani) [inline versions-config/electron-remote-components-win-32.json]: @tejohnso I know, but if we're changing the version in the bucket, having a configuration file that reflects that feels like the right solution.

Either way, it shouldn't change the behavior of the installer, but I'd rather have the version reflected in the config.

:+1:

**Comment by ahmedalsudani** [comment]
:+1: :+1:

**Pull request merged**